client/components/Waitingroom.jsx

```diff
@@ -15,8 +17,9 @@ class Waitingroom extends Component {
     const { socket } = this.props;
     socket.emit('change name', { name: socket.name });
     socket.on('goToGame', () => {
-      this.props.history.push('/imagegame');
+      this.state.game = true;
```

Comment:
this should be this.setState({ game: true })

Reply:
That code is no more.